public/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
There seems to be a CSS issue with the footer.

<img width="428" alt="new" src="https://user-images.githubusercontent.com/42718855/58918556-b056ca00-86f8-11e9-835e-b27a51b5d9fc.png">
<img width="417" alt="old" src="https://user-images.githubusercontent.com/42718855/58918557-b056ca00-86f8-11e9-9a4f-28e26bc0c121.png">